log4j-core/pom.xml

```diff
@@ -444,6 +444,30 @@
             <org.apache.activemq.SERIALIZABLE_PACKAGES>*</org.apache.activemq.SERIALIZABLE_PACKAGES>
           </systemPropertyVariables>
         </configuration>
+        <executions>
+            <execution>
+                <id>default-test</id>
+                <phase>test</phase>
+                <goals>
+                  <goal>test</goal>
+                </goals>
+                <configuration>
+                   <includeJUnit5Engines>junit-jupiter</includeJUnit5Engines>
+                   <forkCount>1C</forkCount>
+                   <runOrder>random</runOrder>
```

Comment:
I've taken to using `<runOrder>balanced</runOrder>` when parallelizing tests. I did set them to random recently in master because I was sick of seeing the same slow tests at the beginning and wanted to change things up. :)

Reply:
I set the run order to random, to empirically check if two test classes do not use the same log files. Hopefully if they do, once in a while we will have some meaningful test failures.

Reply:
Ah, that's another goal of mine whenever I set the ordering to random. I'm trying to see if there are any test interferences that would otherwise have gone unnoticed locally.